docs/080-breaking-changes.rst

```diff
@@ -155,3 +156,15 @@ Interface Changes
   Use the "compact AST" (``--ast-compact--json`` resp. ``AST``) as replacement.
 
 * The old error reporter (``--old-reporter``) has been removed.
+
+
+How to update your code
+=======================
+
+- If you rely on wrapping arithmetic, surround each operation with ``unchecked { ... }``.
+- Optional: If you use SafeMath or a similar library, change ``x.add(y)`` to ``x + y``, ``x.mul(y)`` to ``x * y`` etc.
+- Add ``pragma abicoder v1;`` if you want to stay with the old ABI coder.
+- Change ``byte`` to ``bytes1``.
+- Add intermediate explicit type conversions if required.
```

Comment:
Not sure if we should go into detail here....

Reply:
It still works as a checklist even without the detail but if I were going over it to update my code, having it all listed here would be helpful.

Reply:
I mean the compiler will also generate an error, so I'm not sure such a list is helpful here, especially as it can also only list categories and nothing you can search for in your code.

Reply:
ok. I thought more about listing examples. Categories are indeed less useful.

I wonder if having this whole section is a good idea in general because after looking at details above it I noticed that only the most common cases are listed. I can see how it could still be attractive to people who don't want to spend time reading all of it but this way we're just encouraging more people not to read it :)